hamilton/cli/logic.py

```diff
@@ -152,46 +154,46 @@ def diff_nodes_against_functions(
     )
 
 
-def diff_versions(mapping_v1: Dict[str, str], mapping_v2: Dict[str, str]) -> dict:
+def diff_versions(current_map: Dict[str, str], reference_map: Dict[str, str]) -> dict:
     """Generic diff of two {name: hash} mappings (can be node or origin name)
 
     :mapping_v1: mapping from node (or function) name to its function hash
     :mapping_v2: mapping from node (or function) name to its function hash
     """
-    v1_only, v2_only, edit = [], [], []
+    current_only, reference_only, edit = [], [], []
 
-    for node_name, v1 in mapping_v1.items():
-        v2 = mapping_v2.get(node_name)
+    for node_name, v1 in current_map.items():
+        v2 = reference_map.get(node_name)
         if v2 is None:
-            v1_only.append(node_name)
+            current_only.append(node_name)
             continue
 
         if v1 != v2:
             edit.append(node_name)
 
-    for node_name, v2 in mapping_v2.items():
-        v1 = mapping_v1.get(node_name)
+    for node_name, v2 in reference_map.items():
+        v1 = current_map.get(node_name)
         if v1 is None:
-            v2_only.append(node_name)
+            reference_only.append(node_name)
             continue
 
-    return dict(v1_only=v1_only, v2_only=v2_only, edit=edit)
+    return dict(current_only=current_only, reference_only=reference_only, edit=edit)
```

Comment:
do we need the `only` suffix?

Reply:
since that's what is output to the command line right?

Reply:
I think so for clarity. Thinking in terms of set, "current only" is the difference of "current" and "reference".